operator/pkg/component/component.go

```diff
@@ -398,6 +400,16 @@ func runComponent(c *CommonComponentFields) error {
 	}
 	c.renderer = r
 	c.started = true
+
+	if c.commonValues == "" {
```

Comment:
How can the commonValues shared between different components, seems like we still need to run the translate function for different components with this change?

Reply:
The commonValues are translated from the same IOP with `TranslateHelmValues`:
see: https://github.com/istio/istio/blob/b9c81e74328c18a50924154ed33efb3ccb02fffa/operator/pkg/translate/translate.go#L288
There is no component specific logic with the `TranslateHelmValues` process.

Reply:
yep there is no component specific logic with the TranslateHelmValues process, but with your change it looks like we still need to call translate function for each components? I couldn't understand how the initially translated commonVaues shared between components to avoid duplicate translation work